admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/service/impl/TemplateServiceImpl.java

```diff
@@ -78,6 +83,15 @@
 
 	@Value("#{'${mosip.mandatory-languages}'.concat('${mosip.optional-languages}')}")
 	private String supportedLang;
+	
+	@Value("${mosip.kernel.masterdata.template_idauthentication_event:masterdata/idauthentication_templates}")
+	private String topic;
+	
+	@Value("${websub.publish.url}")
+	private String hubURL;
+	
+	@Value("${mosip.kernel.masterdata.template_idauthentication_event_module_name:ID Authentication}")
+	private String idAuthModuleName;
```

Comment:
this property can be as below:
`@Value("${mosip.kernel.masterdata.template_event_module_names:ID Authentication}")
	private List<String> templateModuleNames;`
	So that it can accept any number of comma separated modules. In future if need to support other modules it will be just config update.

Reply:
yes sir  It was in my mind but they may have more specific need other than module name so it was a unnecessary addition for now. If we get any same request like IDA we will add more predicates but now kept it simple.

Reply:
Ok Urvil..